### DISCUSSION TITLE
Privacy issues on sharing SHA256 key on github

### DISCUSSION THREAD

**Discussion by khandelwalmoksh** [Cyfrin/foundry-full-course-cu#2539] [Q&A] [answered]
Won't there be any privacy issues if we upload our SHA26 key.
 As it is advised to not share it with anybody/anywhere.
 As the developers  might hack/steal the contract funds?

**Comment by mumernisar** [comment]
Which key are you referring to?
Your private key? You should certainly not share it! Although you work on a test wallet in the course on metamask, which you can easily replace by creating a new account, I think. But it is highlighted a lot that it should not be shared so that this thought develops, so you don't 
 accidentally leak your critical information when  working in a real environment with real keys and with funds in it!

**Comment by EngrPips** [comment, marked as answer]
Hello @khandelwalmoksh, Please never have your `private key` in plain text anywhere.
